pkg/agent/config/configmgr_test.go

```diff
@@ -301,6 +301,9 @@ func TestStart(t *testing.T) {
 		}
 		actualErr := mgr.Start(context.TODO())
 		s.Stop()
+		// Ensure all of source queue objects are processed then do check the result action
```

Comment:
There is already a `s.Stop()` in line 304 to wait all queue keys processed,so it's not necessary to re-check and wait here.